.env.example

```
@@ -1,5 +1,17 @@
 SPRING_PROFILES_ACTIVE=local
-DB_URL=jdbc:postgresql://localhost:5432/michelet_db
+
+SERVER_PORT=
+
+DB_URL=
 DB_USERNAME=your_db_username
 DB_PASSWORD=your_db_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_URL key

(UnorderedKey)